docs-guides/source/convert-pytorch-workflow.md

```diff
@@ -24,73 +30,74 @@ import torchvision
+```
+
+## Capture the PyTorch Graph
+
+### TorchScript
+
+[TorchScript](https://pytorch.org/docs/stable/jit.html) is an intermediate representation of a PyTorch model. To generate a TorchScript representation from PyTorch code, use PyTorch's JIT tracer ([`torch.jit.trace`](https://pytorch.org/docs/stable/generated/torch.jit.trace.html)) to _trace_ the model, as shown in the following example:
 
+```python
 # Trace the model with random data.
 example_input = torch.rand(1, 3, 224, 224) 
 traced_model = torch.jit.trace(torch_model, example_input)
-out = traced_model(example_input)
 ```
 
 The process of tracing takes an example input and traces its flow through the model. You can trace the model by creating an example image input, as shown in the above code using random data. To understand the reasons for tracing and how to trace a PyTorch model, see [Model Tracing](model-tracing).
 
-```{admonition} Set the Model to Evaluation Mode
+If your model uses a data-dependent control flow, such as a loop or conditional, the traced model won't generalize to other inputs. In such cases you can experiment with applying PyTorch's JIT script ([`torch.jit.script`](https://pytorch.org/docs/stable/generated/torch.jit.script.html)) to your model as described in [Model Scripting](model-scripting). You can also use a combination of tracing and scripting.
 
-To ensure that operations such as dropout are disabled, it's important to set the model to evaluation mode (_not_ training mode) before tracing. This setting also results in a more optimized version of the model for conversion.
+### ExportedProgram
+
+[ExportedProgram](https://pytorch.org/docs/stable/export.html) is a new intermediate representation of a PyTorch model introduced in PyTorch 2. To generate an ExportedProgram representation from PyTorch code, use PyTorch's exporter ([`torch.export.export`](https://pytorch.org/docs/stable/export.html#torch.export.export)) to _export_ the model, as shown in the following example:
+
+```python
+# Export the model with random data.
+example_inputs = (torch.rand(1, 3, 224, 224),)
+exported_program = torch.export.export(torch_model, example_inputs)
 ```
 
-If your model uses a data-dependent control flow, such as a loop or conditional, the traced model won't generalize to other inputs. In such cases you can experiment with applying PyTorch's JIT script ([`torch.jit.script`](https://pytorch.org/docs/stable/generated/torch.jit.script.html)) to your model as described in [Model Scripting](model-scripting). You can also use a combination of tracing and scripting.
+The process of exporting takes an example input and traces its flow through the model. You can trace the model by creating an example image input, as shown in the above code using random data. To understand the reasons for exporting and how to export a PyTorch model, see [Model Exporting](model-exporting).
```

Comment:
`To understand the reasons for exporting and how to export a PyTorch model, see [Model Exporting](model-exporting)` ---->  remove this line. Dont need it.

Reply:
Removed as suggested